### ISSUE TITLE
a 1d model for the linux community

### ISSUE THREAD

**Issue by jmigueldelgado** [erpas/rgis#45]
Hi. I have been looking for a 1d river model running in linux that could plug into the riverGIS model geometry. Until now I found none. Isn't it time to start writing something like HEC-RAS natively in python? Anybody out there would be interested in joining efforts?

**Comment by erpas** [comment]
Have you tried OpenTelemac with its 1D module, Mascaret [1]? See GNU/Linux installation instructions on [2].
I would love to have something that user-friendly like HEC-RAS as a free software, but writing a model + UI is a huge task and I am not sure there is enough of interest in the community to support the work. And yes, I also mean financial support :-)

[1] http://opentelemac.org/index.php/modules-list/138-mascaret-1-dimensionnal-free-surface-flow-modelling
[2] http://wiki.opentelemac.org/doku.php?id=installation_on_linux

**Comment by jmigueldelgado** [comment]
I agree -- a lot of effort. But given the prices being payed for Mike HYDRO River or SOBEK licenses I would argue that there might be enough interest/money in the community to fund this idea.

Thank you for pointing me to telemac.
